.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
ipfabric only supports 3.7.1+

Reply:
ok, looks like it is tagged only for major releases. The correct python version is defined in the `pyproject.toml`
`python = "^3.7.1"`

Reply:
you could add 3.10 (Nautobot adds support in last release 1.3)

Reply:
yes will add that in another PR.